tests/test_filtering.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+from dolphin import filtering
+
+
+def test_filtering_unwrapped_phase():
```

Comment:
```suggestion
def test_filter_long_wavelegnth():
```

The basic pytests are usually named `test_<name of the function you are testing>`, with more specific tests adding to that function. If we add more to this later, we can also group them into one class with some name like `class TestFilterLongWavelgnth`